docs/spec_draft.md

```diff
@@ -1779,6 +1780,48 @@ More formally, `results[:][result_index] = reduce(input_slices)` where:
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.reduce_precision
+
+### Semantics
```

Comment:
As we do not have a precedent in IEE spec, we need to define all the aspects of the op.  

```
 The input values are rounded to the nearest value representable with the given number of mantissa bits ,using "ties to even" semantics and any values that exceed the range specified by the number of exponent bits are clamped to positive or negative infinity 
```

Reply:
Should NaNs be mentioned?

Reply:
From the [computations](https://github.com/tensorflow/tensorflow/blob/83ccd51d51f47251507405f79f6fe00874d0be2f/tensorflow/compiler/xla/hlo/evaluator/hlo_evaluator_typed_visitor.h#L2096) of the HLO evaluator it seems that we will return back NaN. 

Having said that if we can capture these computations in the spec then we do not have to mention it. Otherwise, we should mention the case of NaN.

Reply:
I don't think we have to explicitly call out NaNs. They seem to follow the proposed formalism below.